deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: vitess-operator
       containers:
       - name: vitess-operator
-        image: planetscale/vitess-operator:v2.4.0
+        image: planetscale/vitess-operator:v2.4.1
```

Comment:
This should also be v2.5.0?

Reply:
No, we bump this to a release tag. Previously pinned here https://github.com/planetscale/vitess-operator/pull/188